content-src/components/Sections/_Sections.scss

```diff
@@ -1,4 +1,30 @@
 .sections-list {
+  &.layout {
+    display: grid;
+    grid-template-columns: repeat(12, auto);
+    grid-row-gap: 10px;
+    grid-column-gap: 10px;
+
+    .full,
+    .left,
+    .right {
+      border: 1px solid $black;
+      padding: 6px;
+    }
+
+    .full {
+      grid-column: 1 / span 12;
+    }
+
+    .left {
+      grid-column: 1 / span 8;
+    }
+
+    .right {
+      grid-column: 9 / span 4;
+    }
+  }
+
```

Comment:
This will be more compatible with the current API layout schema, which assumes you only specify how many column widths a section takes up, and doesn't include a starting position. This is simpler from a definition standpoint (less variables) and can still achieve the same results (and more).